examples/wiznet5k_simpleserver.py

```diff
@@ -31,6 +31,8 @@
 while True:
     conn, addr = server.accept()  # Wait for a connection from a client.
     with conn:
-        data = conn.recv()
-        print(data)
-        conn.send(data)  # Echo message back to client
+        while True:
```

Comment:
Don't nest a `while True` within a `while True`. I do agree that we should only `print(data)` if `data` has been received.

Reply:
Without while, the example will be finished once tcp connection is established. Because recv() function returns 0 immediately as belows. It's not ECHO example, I think. So I added while Ture.

>> Opening W5k Socket, protocol=33
socket connected
Socket read 0
socket_available called with protocol 33
b''
PHY Status: bytearray(b'\xbf')
*** Disconnecting socket #0
*** Closing socket #0

for print(data) opinion, I just refer the example code from this respo.
[](https://github.com/adafruit/Adafruit_CircuitPython_Wiznet5k/blob/master/examples/wiznet5k_simpleserver.py)